### DISCUSSION TITLE
2FA access for many people on the same account

### DISCUSSION THREAD

**Discussion by cruxofcapitalism** [community/community#63916] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello 😊 We are an organisation that have a GitHub Free accoutn and we would like that many people (currently 5) have access to the account. How could we set up the 2FA in this case?

**Comment by dushmanta05** [comment]
Hi!

I think you can use the **Authy** 2FA authenticator app, which supports multiple devices; although I have never tried it personally, they do have this feature, if you can try it out.

Also, I'm just curious: why would you want to have five people access it? You can create an organization and give different roles to each individual or the same roles to each person.

**Comment by oftharsh** [comment]
If you like it please mark it as the answer.

GitHub Free accounts do not offer the ability to enforce Two-Factor Authentication (2FA) at the organization level. However, you can still encourage and guide your organization members to enable 2FA on their individual GitHub accounts. Here's how you can do it:

1. **Educate Members:** Inform your organization members about the importance of enabling 2FA to enhance security. Share resources and guides on how to enable 2FA on GitHub.

2. **Step-by-Step Guide:** Provide a step-by-step guide on how to enable 2FA on GitHub accounts. You can find GitHub's official guide here: [Securing your account with two-factor authentication (2FA)](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).

3. **Recommendations:** Encourage your organization members to use app-based 2FA methods (such as TOTP apps like Google Authenticator or Authy) instead of SMS-based methods, as app-based methods are generally more secure.

4. **Reminders:** Send regular reminders to your organization members to enable 2FA if they haven't already. You can use internal communication channels, email, or other methods to ensure everyone is aware.

5. **Settings Enforcement (Paid Plans):** If you upgrade to a paid plan, such as GitHub Team or GitHub Enterprise, you will have more control over security settings. These plans allow you to enforce 2FA for organization members.

Remember that while you can encourage and guide your members to enable 2FA, it's ultimately up to each individual member to set up and manage their own 2FA settings on their GitHub accounts.
